src/mlpack/bindings/julia/print_jl.cpp

```diff
@@ -168,8 +169,13 @@ void PrintJL(const util::ProgramDoc& programInfo,
   cout << endl;
 
   // Next print the description.
-  cout << util::HyphenateString(programInfo.documentation(), 0) << endl;
-
+  cout << util::HyphenateString(longDesc.longDescription(), 0) << endl;
```

Comment:
For the CLI bindings, you had put two line endings after the long description and between each example.  Should we do the same here?

Reply:
I have added `endl` after this. 👍

Reply:
Thanks!